crates/collab/k8s/collab.template.yml

```diff
@@ -109,17 +109,17 @@ spec:
                 secretKeyRef:
                   name: zed-client
                   key: checksum-seed
-            - name: LIVE_KIT_SERVER
+            - name: LIVEKIT_SERVER
               valueFrom:
                 secretKeyRef:
                   name: livekit
                   key: server
-            - name: LIVE_KIT_KEY
+            - name: livekit_KEY
```

Comment:
This environment variable name looks incorrect.

Reply:
Good catch!